.github/workflows/continuous-deployment.yml

```diff
@@ -174,7 +194,8 @@ jobs:
           -bf results-latest.json \
           -jf results.json \
           -fjf filtered_results.json \
-          --file cams.zip
+          --file cams.zip \
+          --app_id ${{ secrets.VERACODE_APP_ID }}
```

Comment:
What does this addition do?

Reply:
this was used to see if this would trigger an upload, but it did not. Just didn't clean it up.